Functions.Templates/Templates/EdgeHubTrigger-CSharp/function.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
This file is not needed either. This will be auto generated by the `Microsoft.NET.Sdk.Functions` package